Assets/MixedRealityToolkit-SDK/Features/Boundary/Components/Scripts/PlayAreaVisualizer.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.BoundarySystem;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Boundary
+{
+    /// <summary>
+    /// Visualizes the inscribed rectangular play area within the configured boundary.
+    /// </summary>
+    public class PlayAreaVisualizer : MonoBehaviour
+    {
+        [Tooltip("Should the play area be visualized?")]
+        [SerializeField]
+        private bool isPlayAreaVisualized = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the play area is / should be visualized.
+        /// </summary>
+        public bool IsPlayAreaVisualized
```

Comment:
Shouldn't the Boundary System be keeping track of this?

Reply:
no. it's up to the app. they may want to show/hide based on specific scenarios

Reply:
yeah, we should let the boundary manager handle this.